docker/post_github_comment.py

```diff
@@ -31,13 +31,14 @@ def get_docker_image_size(docker_image, is_contribution: bool = False) -> str:
                 res.raise_for_status()
                 size_bytes = res.json()['images'][0]['size']
                 size = '{0:.2f} MB'.format(float(size_bytes)/1024/1024)
+                break
             except Exception as ex:
                 print("[{}] failed getting image size for image: {}. Err: {}".format(i, docker_image, ex))
```

Comment:
Maybe?
(btw, this whole loop feel cumbersome.)
```suggestion
                print("Attempt [{}] failed getting image size for image: {}. Err: {}".format(i, docker_image, ex))
```